types/lh.d.ts

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
is `lh.d.ts` preferred over `lighthouse.d.ts`?

Reply:
I don't have a string preference. I thought `lh.d.ts` made more sense because we refer to this stuff via the `LH` global.